### DISCUSSION TITLE
Customize table without Quasar

### DISCUSSION THREAD

**Discussion by Ottokranz** [zauberzeug/nicegui#3317] [Q&A] [answered]
### Question

I'm trying to customize a table cell only with nicegui elements. For example if I have this:
```py
table.add_slot('body-cell-Inhalt', r'''
    <q-td key="id" :props="props">
        <div class="self-center">{{ props.row.templateContent }}</div>
    </q-td>
''')
```
I want to recreate it like that:
```py
with table.add_slot('body-cell-Inhalt'):
    ui.label().bind_text(props.row.templateContent)
```
How can I access the data in each row? Is that even possible or do I have to use Quasar to customize my table?

**Comment by falkoschindler** [comment, marked as answer]
Hi @Ottokranz,
That's currently not possible (see #846, #1138, #2080, #2625, and #3148).
But there's PR #3261 trying to implement this feature.
